dashboard/test/models/concerns/pd/form_test.rb

```diff
@@ -91,14 +91,6 @@ class Pd::FormTest < ActiveSupport::TestCase
     refute form.valid?
   end
 
-  test 'pd form does not check required fields if status is incomplete' do
```

Comment:
This is the most awkward part of using this approach ... to test, we'd need to create an application with a status, rather than keep it as a form because a form doesn't have a status. Similar tests already exist in TeacherApplicationTest, so rather than duplicate here, I didn't include a test.